packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js

```diff
@@ -268,7 +270,7 @@ const CollectionTypeFormWrapper = ({ allLayoutData, children, slug, id, origin }
         // Enable navigation and remove loaders
         dispatch(setStatus('resolved'));
 
-        replace(`/content-manager/collectionType/${slug}/${data.id}${rawQuery}`);
+        replace(`/content-manager/collectionType/${slug}/${data.id}`);
```

Comment:
I think this still needs the query otherwise when you edit a non-default locale you get redirected to the default locale :)